### DISCUSSION TITLE
How do I pass additional compiler flags to poac build command?

### DISCUSSION THREAD

**Discussion by shyamsundarb-arch** [cabinpkg/cabin#940] [Q&A] [answered]
I want to use the [mold](https://github.com/rui314/mold) linker. I have it installed in my machine. For the compiler to use the mold linker, I need to pass ```-fuse-ld=mold``` as a flag to the compiler. With poac how do that?

**Comment by yaito3014** [comment, marked as answer]
You can either:
- use `--run` option in mold like `mold --run poac build`, or
- set `LDFLAGS` environment variable to `-fuse-ld=mold`, or
- add these line to poac.toml (this lets clang produce lots of warning though)
  ```toml:poac.toml
  [profile]
  cxxflags = ["-fuse-ld=mold"]
  ```

There seems to be no other way, unfortunately.

**Comment by shyamsundarb-arch** [comment]
Thank you @yaito3014
